src/ManageCourses.Api/Controllers/AcceptTermsController.cs

```diff
@@ -17,7 +18,7 @@ public AcceptTermsController(IManageCoursesDbContext context)
             this.context = context;
         }
 
-        [Authorize]
+        [Authorize(AuthenticationSchemes = BearerTokenDefaults.AuthenticationScheme)]
```

Comment:
Can we create a class that inherits from `Authorize` ie `BearerTokenAuthorize` and its counterpart ``BearerTokenApiKeyAuthorize``

So it shows up as a hard choice made therefore usage of `Authorize` can be quickly discovered quickly

Reply:
done. good call :+1: